ToDoTakeTWo/app/src/main/java/com/korbkenny/todotaketwo/MainActivity.java

```diff
@@ -0,0 +1,197 @@
+package com.korbkenny.todotaketwo;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.BaseAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity {
+    private ListView mListView;
+    private Button mAddNewList;
+    final Context context = this;
+    public ArrayAdapter<ToDoList> mArrayAdapter;
+    public BigOldListOfLists bigOldListOfLists;
```

Comment:
Remember to be consistent with your naming - either use the "m" prefix on ALL member variables (that's what we recommend) or on NONE of them. Consistency is really important, especially when working with teammates.